tests/components/powerwall/test_config_flow.py

```diff
@@ -114,3 +138,86 @@ async def test_form_wrong_version(hass):
 
     assert result3["type"] == "form"
     assert result3["errors"] == {"base": "wrong_version"}
+
+
+async def test_already_configured_(hass):
+    """Test we abort when already configured."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+
+    config_entry = MockConfigEntry(
+        domain=DOMAIN, data={CONF_IP_ADDRESS: "1.1.1.1"}, source="ignore"
```

Comment:
Should the existing entry have source ignore?

Reply:
Nope. Good catch on the copy pasta.

Thanks